packages/nocodb/src/db/IBaseModelSqlV2.ts

```
@@ -111,7 +219,72 @@ export interface IBaseModelSqlV2 {
+
+  validate(
+    data: Record<string, any>,
+    columns?: Column[],
+    {
+      typecast,
+      allowSystemColumn,
+    }?: { typecast?: boolean; allowSystemColumn?: boolean },
+  ): Promise<boolean>;
+
+  runOps(ops: Promise<string>[], trx?: CustomKnex): Promise<void>;
+  statsUpdate(_args: { count: number }): Promise<void | any>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)